{"name": "test_delete_pet", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 200\n +  where 404 = <Response [404]>.status_code", "trace": "def test_delete_pet():\n        path = \"v2/pet/1989\"\n        response = requests.delete(url=baseUrl+path)\n        logging.info(f'Delete method is called')\n        logging.info(f'Response: {response.text}')\n>       assert response.status_code == 200\nE       assert 404 == 200\nE        +  where 404 = <Response [404]>.status_code\n\ntests\\test_pets.py:55: AssertionError"}, "attachments": [{"name": "log", "source": "07fa51e7-3d98-49c8-9ca3-691c247477d5-attachment.txt", "type": "text/plain"}], "start": 1658400321858, "stop": 1658400322433, "uuid": "619082ac-b7dd-4c1e-9cb5-e9707cd80953", "historyId": "c85a1c416cd6f101f10d4112a3fa520d", "testCaseId": "08eb1e629bf2e475663da0bdfad333e5", "fullName": "tests.test_pets#test_delete_pet", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_pets"}, {"name": "host", "value": "A00639991A"}, {"name": "thread", "value": "31176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_pets"}]}